test-integ/catalogv2/simple_test.go

```diff
@@ -0,0 +1,335 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package catalogv2
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+	"testing"
+
+	pbauth "github.com/hashicorp/consul/proto-public/pbauth/v2beta1"
+	"github.com/hashicorp/consul/proto-public/pbresource"
+	libassert "github.com/hashicorp/consul/test/integration/consul-container/libs/assert"
+	"github.com/hashicorp/consul/test/integration/consul-container/libs/utils"
+	"github.com/hashicorp/consul/testing/deployer/sprawl"
+	"github.com/hashicorp/consul/testing/deployer/sprawl/sprawltest"
+	"github.com/hashicorp/consul/testing/deployer/topology"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/hashicorp/consul/test-integ/topoutil"
+)
+
+var (
+	dev1_17Images = topology.Images{
+		ConsulCE: "hashicorppreview/consul:1.17-dev",
+		// ConsulEnterprise: "hashicorppreview/consul-enterprise:1.17-dev",
+		ConsulEnterprise: "consul-dev:latest",
+		Dataplane:        "hashicorppreview/consul-dataplane:1.3-dev",
+	}
+)
+
+var flagUseDevImages = flag.Bool("dev-images", false, "set to enable 1.17 dev images for everything")
+
+// TODO(rb): bump the testing/deployer default versions when the various components have
+// released versions
+func getImages() topology.Images {
+	if *flagUseDevImages {
+		return dev1_17Images
+	}
+	images := utils.TargetImages()
+	images.Dataplane = dev1_17Images.Dataplane
+	return images
+}
+
+func TestSimpleTopologySuite(t *testing.T) {
+	suite.Run(t, new(SimpleTopologySuite))
+}
+
+type SimpleTopologySuite struct {
```

Comment:
I'd like a better name that describes a least a little bit what it does. Also maybe a summary, considering this is about 300 lines

Reply:
Renamed file and added a description of the test setup.